prediction_market_agent_tooling/markets/agent_market.py

```diff
@@ -198,6 +205,36 @@ def get_binary_markets(
     def get_binary_market(id: str) -> "AgentMarket":
         raise NotImplementedError("Subclasses must implement this method")
 
+    @staticmethod
+    def redeem_winnings(api_keys: APIKeys) -> None:
+        """
+        On some markets (like Omen), it's needed to manually claim the winner bets. If it's not needed, just implement with `pass`.
+        """
+        raise NotImplementedError("Subclasses must implement this method")
+
+    @staticmethod
+    def get_trade_balance(api_keys: APIKeys) -> float:
+        """
+        Return balance that can be used to trade on the given market.
+        """
+        raise NotImplementedError("Subclasses must implement this method")
+
+    @staticmethod
+    def verify_operational_balance(api_keys: APIKeys) -> bool:
+        """
+        Return if the user has enough of operational balance, if not needed, just implement with `pass`.
```

Comment:
Function return type is a boolean
```suggestion
        Return `True` if the user has enough of operational balance. If not needed, just return `True`.
```

Reply:
Oops True! 😄